### PR TITLE
Experience/10911/manage connection 2

### LINKED ISSUE #10911: Website Redesign - Managing your connection
As a User,
I want to visit the ReportStream **Managing your connection** Page,
So that I can become informed about the tools available to me to manage my account

Designs
- Figma Link: https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design

Content Updates
- [See Figma Link](https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design)

Page Title
- "CDC ReportStream - Managing your connection" 

Analytics
- TBD 

Storybook updates
- TBD

Verification Steps
- Signed off in Staging by Product / Design
- Signed off in Production by Product / Design / Engineering

Acceptance Criteria:
- [ ] Analytics / Metrics Required - TBD
- [ ] Storybook update required - n/a
- [ ] Unit Tests - n/a
- [ ] End to End Tests - TBD
- [x] Manual Testing Steps

Dependencies:
- Designs - https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=1267-6590&mode=design
- Platform API Endpoints - N/A
- Links - TBD

Additional Details:
- The deployment strategy will follow the general deployment strategy for the Website Redesign EPIC. We will deliver this page in isolation from the other pages. 
- Conversion to MDX will be determined by the Engineer owning this ticket

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

ReportStream Managing your connection Page

Test Steps:
1. Log into RS
2. navigate to /manage-connection

## Changes
- Added quicklinks
- Fixed broken links
- style updates

Screenshot 2023-08-30 at 6 17 28 PM

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #10911

### CODE REVIEW

File: frontend-react/src/shared/Card/Card.module.scss [hunk 1]
```diff
@@ -26,3 +26,8 @@
         border-radius: 0;
     }
 }
+
+:global(.usa-card__container) {
```
Comment (penny-lischer) [inline frontend-react/src/shared/Card/Card.module.scss:30]: @jpandersen87 do you see any issues with me overriding these styles?

Comment (jpandersen87) [reply frontend-react/src/shared/Card/Card.module.scss:30]: no, but it should go into the uswds overrides sheet instead

**Review by jpandersen87 — CHANGES_REQUESTED** [review]

File: frontend-react/src/shared/Card/Card.module.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (etanb) [reply frontend-react/src/shared/Card/Card.module.scss:30]: don't need so many `:global` declarations.

**Comment by audreykwr** [comment]
- [ ] Make quick links sentence case - "Quick links"
- [ ] Contact us links in each card as well as the submit service request at the bottom need to be updated to [this link](https://app.smartsheetgov.com/b/form/da894779659b45768079200609b3a599) per Slack thread

**Comment by audreykwr** [comment]
Thanks @penny-lischer! Still missing one link fix at the "submit service request" link

**Review by jpandersen87 — APPROVED** [review]

**Review by jillian-hammer — APPROVED** [review]
looks great, thanks for making those adjustments @penny-lischer!

**Review by etanb — APPROVED** [review]

**Review by audreykwr — APPROVED** [review]

**Review by jillian-hammer — APPROVED** [review]

**Pull request merged**